app/src/Components/ListTable/ListRowButtons/ExecuteRowButton.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Mutation } from 'react-apollo';
+import IconButton from '@material-ui/core/IconButton';
+import PlayArrow from '@material-ui/icons/PlayArrow';
+import Snackbar from '@material-ui/core/Snackbar';
+
+
+class ExecuteRowButton extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            'open': false
+        };
+    }
+
+    handleSnackbarClose = () => this.setState({ 'open': false });
+
+    execute = (func) => {
+        func();
+        this.setState({ 'open': true });
+    }
+
+    render() {
+        return (<Mutation
+            mutation={this.props.parameter.repository.execute()}
+            variables={{ 'id': this.props.recordId }}>
+            {(executeFunc, params) => {
+                const { loading, error, called, data } = params;
+                if (loading) {
+                    return <p>Loading...</p>;
+                } else if (called && data) {
```

Comment:
Isn't the else part of the if redundant as we return in case of error?
```suggestion
                }
                if (error) {
                    return <p>Error...</p>;
                }
                if (called && data) {
```